### DISCUSSION TITLE
phantom branch limiting concurrency

### DISCUSSION THREAD

**Discussion by djmitche** [renovatebot/renovate#8437] [Request Help]
From a run just a few minutes ago, which opened 1 PR, I see
```
DEBUG: 0 PRs are currently open
DEBUG: PR concurrent limit remaining: 2
DEBUG: Calculated maximum PRs remaining this run
{
  "prsRemaining": 2
}
DEBUG: PullRequests limit = 2
DEBUG: Calculating branchConcurrentLimit (2)
DEBUG: 1 branches are currently open
DEBUG: Branch concurrent limit remaining: 1
DEBUG: Calculated maximum branches remaining this run
{
  "branchesRemaining": 1
}
DEBUG: Branches limit = 1
```

There's lots of logging before the `0 PRs are currently open` line indicating how it figured that out (and that determination is correct -- the latest PR had been merged before this run).

But, there's no logging about how renovate has determined `1 branches are currently open`.  I see three open branches with the word "renovate" in https://github.com/taskcluster/taskcluster/branches/all?query=renovate

![image](https://user-images.githubusercontent.com/28673/106012623-9619ee80-6089-11eb-8015-680233e059a4.png)

which of those is the issue?  Is it the eslint branch?  Why didn't renovate delete that branch itself?

I'll try deleting that branch to see if it helps, but if that is the issue then some extra information in the logs would be helpful, and even moreso if that logging could indicate that a branch was found associated with a merged PR.

**Comment by rarkins** [comment]
@zharinov can you take a look?

**Reply by zharinov** [reply]
Does this problem happen in periodic way? Or it happened just once?

**Comment by djmitche** [comment]
It, or something very much like it, has happened three times recently.  In the other cases (https://github.com/taskcluster/taskcluster/pull/4407, https://github.com/taskcluster/taskcluster/pull/4395) the existence of a closed PR left later updates to those same packages in the "blocked" category, and renaming the PRs fixed the issue.  I'm not sure if that's the same problem or not.
